tests/unit/extensions/test_kde_neon.py

```diff
@@ -53,7 +53,9 @@ def kde_neon_extension_with_default_build_snap_from_latest_edge():
         yaml_data={
             "base": "core22",
             "parts": {
-                "part1": {"build-snaps": ["kf5-5-105-qt-5-15-9-core22-sdk/latest/edge"]}
+                "part1": {
+                    "build-snaps": ["kf5-5-108-qt-5-15-10-core22-sdk/latest/edge"]
```

Comment:
side note, you don't need to change this everytime, this is to test that manually changing the default is possible